source/gui/addonGui.py

```diff
@@ -60,20 +60,25 @@ def __init__(self,parent):
 		self.helpButton.Disable()
 		self.helpButton.Bind(wx.EVT_BUTTON,self.onHelp)
 		entryButtonsSizer.Add(self.helpButton)
+		# Translators: The label for a button in Add-ons Manager dialog to enable or disable the selected add-on.
+		self.toggleButton=wx.Button(self,label=_("Disable add-on"))

```

Comment:
Rename the toggleButton variable to enableDisableButton or something similar. ToggleButton I feel is not clear enough...